MetaMask/metamask-extension
# feat: Upgrade assets controllers to 43 with multichain polling for token lists + detection

## **Description**

This upgrades assets controllers to version 43.  It allows us to poll for token lists and token detection across chains.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/28447?quickstart=1)

## **Related issues**

Fixes:

## **Manual testing steps**

1.  Onboard to metamask
2. Verify your erc20 tokens are detected
3. Click a token
4. Verify data on token details page
5. Switch networks
6. Repeat steps 2-4

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
